docs/upload/portal.md

```diff
@@ -1,23 +1,32 @@
-# WiFi and MQTT configuration
+# Wifi and MQTT configuration
```

Comment:
I value consistency a lot. You sometimes use "Wifi" and sometimes "WiFi". Wikipedia suggests you should use "Wi-Fi" but I personally don't care as long as it's spelled the same consistently.

Reply:
I change all inconsistent ones as well whenever I spot them in a document I'm making changes in anyway - generally to **WiFi**, e.g.

https://github.com/1technophile/OpenMQTTGateway/commit/ba1877fc3010c8cc030731770af03d5f2f7ff269

as I value consistency a lot as well ;)